bootstrap/bootstrap.go

```diff
@@ -219,16 +199,39 @@ func (b *Bootstrap) StartAPIServer(ctx context.Context) error {
 		nil,
 	)
 
+	accountKeys := make([]*requester.AccountKey, 0)
+	account, err := b.client.GetAccount(ctx, b.config.COAAddress)
+	if err != nil {
+		return fmt.Errorf(
+			"failed to get signer info account for address: %s, with: %w",
+			b.config.COAAddress,
+			err,
+		)
+	}
+	signer, err := createSigner(ctx, b.config, b.logger)
```

Comment:
**Note:** After viewing this [comment](https://github.com/onflow/flow-go/blob/master/integration/benchmark/account/account.go#L29), I am not entirely sure if we should have one `crypto.Signer` object for each account key, or create one for each account key. Are there any thread-safe concerns here? :thinking:

Reply:
A single signer cannot be used across go routines, so we need multiple. One for every account key.

Actually the `crypto.PrivateKey` is not thread safe either! So if multiple account keys have the same private key, you need to create multiple copies of `crypto.PrivateKey`.

Reply:
Note that we have 2 types of signers, one is the in-memory signer:
1. `crypto.NewInMemorySigner(config.COAKey, crypto.SHA3_256)`
2. The other is `requester.NewKMSKeySigner`, which doesn't deal at all with `crypto.PrivateKey`

Currently, only 2. is supposed to be used in production. And I'm not sure if it has any thread-safety concerns, as it uses Cloud KMS for signing.

1. is supposed to be used for local development / testing.